packages/checkbox/src/Checkbox.tsx

```diff
@@ -313,11 +322,13 @@ export default class Checkbox extends PureComponent<CheckboxProps> {
           {...checkboxWrapper.prop}
           className={cx(wrapperStyle, {
             [wrapperStyleChecked]: checked && !indeterminate && !disabled,
+            [wrapperStyleAnimated]: animate && !indeterminate && !disabled,
           })}
         >
           <div
             className={cx(checkboxStyle, checkboxBackgroundImage, {
               [checkboxStyleChecked]: checked && !indeterminate && !disabled,
+              [checkboxStyleAnimated]: animate && !indeterminate && !disabled,
```

Comment:
I don't think this needs the indeterminate or disabled checks to work properly. Ditto for the other instance.

Reply:
it's needed for this one because if we go from `checked -> checked + indeterminate -> checked`, we'll animate the check, which looks pretty weird. I kept it for the wrapper just for symmetry.

Reply:
Makes sense! 👍